### PR TITLE
add phone number to countries

### PR DESCRIPTION
**Pull request by cedvdb**

### CODE REVIEW

**Comment by cedvdb** [comment]
I just saw you were asking for a PR for this in the issues. So I did it fast. if you want to add other things I just copy pasted the info from [here ](https://gist.github.com/Goles/3196253) and ran a forEach loop.

```
codes.forEach(code => {
   const found = phones.find(p => p.Iso2 === code.alpha2Code);
   if (found) {
      code.dial = found.Dial;
   }
});
```

File: projects/angular-material-extensions/select-country/src/lib/db.ts [hunk 1]
```diff
@@ -1,1224 +1,1426 @@
 import {Country} from './mat-select-country.component';
 
 export const COUNTRIES_DB: Country[] =
```
Comment (AnthonyNahas) [inline projects/angular-material-extensions/select-country/src/lib/db.ts:3]: @cedvdb may you check the latest version `3.0.1` and adapt your code into the multiple i18n dbs ?

Thank you for your work

**Review by AnthonyNahas — CHANGES_REQUESTED** [review]

**Comment by essana3** [comment]
@cedvdb the gist you're using as reference for calling codes has some mistakes.

**Comment by essana3** [comment]
@AnthonyNahas I think this should be closed since it's already been added in 4.1.0

**Pull request closed without merging**